monitors/SyntheticPegMonitor.js

```diff
@@ -83,6 +99,102 @@ class SyntheticPegMonitor {
     }
   };
 
+  // Checks difference between minimum and maximum historical price over `volatilityWindow` amount of time.
+  // Fires a message if the difference exceeds the `volatilityAlertThreshold` %.
+  checkPegVolatility = async () => {
```

Comment:
Is there a way to disable one or both of these volatility checks using the config? This may not apply to this PR, but might affect how it's integrated into the monitor `index.js` file.

I think for all of the monitoring checks, it would be really nice to be able to turn off the alerts in case we decide that monitor is too noisy or isn't useful.

Reply:
the root monitors/index.js can choose which volatility checker to call, is that what you mean?

Reply:
We should probably do this in a separate PR -- I think we should be able to config the monitor bot to turn off certain monitors. Not necessary short term. Just a nice-to-have.

Reply:
> > @chrismaree How should I test slack alerts are sent properly?
> > @mrice32 I'm curious how you think this volatility calculation would work with the Cryptowatch/Uniswap pricefeeds in production?
> 
> I think this will work fine since no requests are actually made during on each historical price lookup. My only worry is that the lookback loop may get slow if there are a lot of price changes and the price feed is computing a TWAP for each one or something. But I could only see that happening with a long lookback, a long TWAP, and many many price updates, so not really something we should spend a lot of time worrying about now.

I've actually seen this issue occur randomly. A 3600 second look back performed fine for most of my testing and then suddenly the `getHistoricalPrice()` at a 300+ second lookback started to hang. There are several solutions:
- Keep volatility windows small.
- During lookback, instead of querying a price for every second, sample a price for every N seconds.
- Implement a timeout that will exit the function if the lookback cannot be completed